### PR TITLE
Rename JsbBridgeWrapper API

### PR DESCRIPTION
**Pull request by timlyeee**

### CODE REVIEW

File: cocos/platform/android/java/src/com/cocos/lib/JsbBridgeWrapper.java [hunk 1]
```diff
@@ -28,42 +28,62 @@ of this software and associated engine source code (the "Software"), a limited,
 import java.util.HashMap;
 
 public class JsbBridgeWrapper {
-    public interface JavaCallback {
-        void onTrigger(String arg);
+    //Interface for listener, should be implemented and dispatched
+    public interface onNativeEventListener {
```
Comment (minggo) [inline cocos/platform/android/java/src/com/cocos/lib/JsbBridgeWrapper.java:32]: onNativeEventListener -> OnNativeEventListener。接口和类名一样，首字母要大写。

File: cocos/platform/android/java/src/com/cocos/lib/JsbBridgeWrapper.java [hunk 2]
```diff
@@ -28,42 +28,62 @@ of this software and associated engine source code (the "Software"), a limited,
...
+    public void removeAllListenersForEvent(String eventName) {
+        this.eventMap.remove(eventName);
     }
-
-    public void dispatchScriptEvent(String event, String arg0) {
-        JsbBridge.sendToScript(event, arg0);
+    /**
+     * Remove all event registered. Use it carefully!
+     */
+    public void removeAllEvents() {
```
Comment (minggo) [inline cocos/platform/android/java/src/com/cocos/lib/JsbBridgeWrapper.java:73]: removeAllEvents  -> removeAllListeners

File: cocos/platform/android/java/src/com/cocos/lib/JsbBridgeWrapper.java [hunk 3]
```diff
@@ -28,42 +28,62 @@ of this software and associated engine source code (the "Software"), a limited,
 import java.util.HashMap;
 
 public class JsbBridgeWrapper {
-    public interface JavaCallback {
-        void onTrigger(String arg);
+    //Interface for listener, should be implemented and dispatched
+    public interface OnNativeEventListener {
```
Comment (minggo) [inline cocos/platform/android/java/src/com/cocos/lib/JsbBridgeWrapper.java:32]: OnNativeEventListener -> OnScriptEventListener

File: cocos/platform/android/java/src/com/cocos/lib/JsbBridgeWrapper.java [hunk 4]
```diff
@@ -28,42 +28,62 @@ of this software and associated engine source code (the "Software"), a limited,
...
+        void onNativeEvent(String arg);
```
Comment (minggo) [inline cocos/platform/android/java/src/com/cocos/lib/JsbBridgeWrapper.java:33]: onNativeEvent -> onScriptEvent

**Review by dumganhar — APPROVED** [review]

**Review by minggo — APPROVED** [review]

**Pull request merged**